model-server-test/src/test/kotlin/RepositoriesManagerWithDatabaseTest.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+import org.modelix.model.lazy.RepositoryId
+import org.modelix.model.server.handlers.RepositoriesManagerTest
+import org.modelix.model.server.store.IgniteStoreClient
+import java.sql.DriverManager
+import kotlin.test.AfterTest
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class RepositoriesManagerWithDatabaseTest : RepositoriesManagerTest(IgniteStoreClient()) {
```

Comment:
This lacks some corner cases:

* Deleting a repo without any data
* Tests for error handling when a query fails